framework/src/modules/chain/modules/transport.js

```diff
@@ -216,15 +216,16 @@ __private.receiveTransactions = function(
  * @todo Add description for the params
  */
 __private.receiveTransaction = function(
-	transaction,
+	transactionJSON,
 	nonce,
 	extraLogMessage,
 	cb
 ) {
-	const id = transaction ? transaction.id : 'null';
+	const id = transactionJSON ? transactionJSON.id : 'null';
```

Comment:
should it still insert `null`?

Reply:
It was working before and I didn't feel the need to change the functionality here.